### PR TITLE
Add login link in Forgot Password Model

### LINKED ISSUE #3110: No way to go back when you click Forget Password
**Actual Behaviour**
When user clicks *Forget Password* in login model, there's no way of coming back to login instead of closing the model. A user has to close model if he want to login.
**Expected Behaviour**
There should be a link below the button. By clicking the button, login model should open again.

**Steps to reproduce it**
Add a login link that open login model again without exit the existing model

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by MuhammadWasif**
Fixes #3110 
No errors  ✅
Following best practices 🚀
Passed all tests 😃
Screenshots of the change: 
https://drive.google.com/file/d/1PKIrBgvkiDx8gS9TkKbjHvO-Q-SpOWHM/view

From above video, changes can bee seen in action.

### CODE REVIEW

File: src/components/Auth/ForgotPassword/ForgotPassword.react.js [hunk 1]
```diff
@@ -136,6 +142,13 @@ class ForgotPassword extends Component {
           >
             {loading ? <CircularProgress size={24} /> : 'Reset'}
           </Button>
+          <LinkContainer>
+            <StyledLink
+              onClick={() => actions.openModal({ modalType: 'login' })}
+            >
+              <Translate text="Login Instead!" />
```
Comment (akshatnitd) [inline src/components/Auth/ForgotPassword/ForgotPassword.react.js]: Changes looks good! Please change the text to `Back to Login`.

Comment (MuhammadWasif) [reply src/components/Auth/ForgotPassword/ForgotPassword.react.js]: Oh.....sound good! 
and done  ✅

**Comment by NishithP2004** [comment]
LGTM

**Comment by MuhammadWasif** [comment]
> LGTM

Please merge

**Comment by MuhammadWasif** [comment]
> Please update your branch

I did! Please check now

**Comment by MuhammadWasif** [comment]
@mariobehling Here's the deployment link: 
https://sad-visvesvaraya-f4a745.netlify.com/
1PKIrBgvkiDx8gS9TkKbjHvO-Q-SpOWHM_e_download_gd_true

**Comment by mariobehling** [comment]
Good job! Thanks.

**Pull request merged**